src/dedicated/support/ps.md

```diff
@@ -0,0 +1,68 @@
+# Differences from Platform.sh on the Grid
+
+When using Platform.sh Dedicated, a few configuration options and tools function differently from Platform.sh on the Grid, aka the Development Environment.
+
+## PHP
+
+Platform.sh Dedicated comes with `pdo`, `apcu`, `curl`, `gd`, `imagick`, `ldap`, `mcrypt`, `mysqli`, `redis`, `soap`, and `opcache` extensions enabled by default.
+
+In addition, we can enable `enchant`, `gearman`, `geoip`, `gmp`, `http`, `pgsql`, `pinba`, `pspell`, `recode`, `tidy`, `xdebug`, `oci8` (PHP5.6 only), or any extension with a pre-existing package in the Debian Apt repository if desired.  Please request such extensions via a ticket.
+
+Custom `php.ini` files are not supported on Platform.sh Dedicated. However, all php options that can be changed at runtime are still available. For example the memory limit can be changed using `ini_set('memory_limit','1024M');`
+
+PHP options that can we can change via support ticket include: 
+
+* `max_execution_time`
+* `max_input_time`
+* `max_input_vars`
+* `memory_limit`
+* `post_max_size`
+* `request_order`
+* `upload_max_filesize`
+
+### XDebug
+
+Platform.sh runs a second PHP-FPM process on all Dedicated clusters that has [Xdebug](https://xdebug.org/) enabled, but is only used if a request includes the appropriate Xdebug header.  That means it's safe to have XDebug "always on", as it will be ignored on most requests.
+
+To obtain the key you will need to file a ticket to have our support team provide it for you.  Staging and Production have separate keys.  Set that key in the Xdebug helper for your browser, and then whenever you have Xdebug enabled the request will use the alternate development PHP-FPM process with Xdebug.
+
+## Cron tasks may be interrupted by deploys
+
+On Platform.sh Grid projects, a running cron task will block a deployment until it is complete.  On Platform.sh Dedicated, however, a deploy will terminate a running cron task.
+
+Specifically, when a deploy to either Production or Staging begins any active cron tasks are send a SIGTERM message so that they can terminate gracefully if needed.  If they are still running 2 seconds later a SIGKILL message will be sent to forceably terminate the process.
```

Comment:
```suggestion
Specifically, when a deploy to either Production or Staging begins, any active cron tasks are sent a `SIGTERM` message so that they can terminate gracefully if needed.  If they are still running 2 seconds later a `SIGKILL` message will be sent to forcibly terminate the process.
```